google/ggrc-core
# GGRC-4445 Cover 'create' permissions with tests cases where Global Editor is Workflow Admin

# Dependencies

This PR is `on hold` until the dependencies are merged:

- [x] #7299 


# Issue description

Global Editor & WF Admin missed 'Create' permissions.

# Steps to test the changes

See [RBAC doc](https://docs.google.com/spreadsheets/d/1bUarGC6Pe3mRfWrsTF1E69AYA4gAYh-D1xfEhWl4LeM/edit#gid=622799523) for more details.

# Solution description

#7299 Added hot-fix to provide 'Create' permission to Global Editor as Workflow Admin.
Here is integration tests for #7299 fix.

# Sanity checklist




# PR Review checklist
